README.md

```diff
@@ -44,6 +44,18 @@ Stop geOrchestra with
 docker-compose down
 ```
 
+## SSL/TLS
+
+This repo comes with a self signe cert which is _not_ valid. To test with a valid cert you can
+install [mkcert](https://github.com/FiloSottile/mkcert) and do the following:
```

Comment:
where should it be installed ? Sorry if it sounds obvious ;-)

Reply:
you mean `mkcert` ? I though that the person who wants to use this will find his distro/OS and install it the way they want.

Reply:
Yes, so, I would say "you can
install mkcert on your host and do the following:"